package/psyneulinkviewer/node.py

```diff
@@ -0,0 +1,42 @@
+import sys
+import subprocess
+import logging
+import configuration
+from setuptools.command.install import install
+from packaging.version import Version
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+def install_node():
+    logging.info("Installing node")
+    subprocess.run(configuration.conda_forge, shell=True)
+    subprocess.run(configuration.node_installation, shell=True)
+
+def check_node_installation():
+    node_version = None
+    try:
+        node_version = subprocess.run(
+            ["node", "--version"],
+            capture_output = True,
+            text = True 
+        ).stdout
+        logging.info("Node version detected : %s", node_version)
+    except Exception as error:
+        logging.error("Error with node installation: %s ", error)
+
+    if node_version is None:
```

Comment:
we move to this logic only if the exception is FileNotFoundError, otherwise for other exceptions/errors we log the error and we stop the installation.